docs/user/openstack/etcd-ephemeral-disk.md

```diff
@@ -0,0 +1,203 @@
+# Moving etcd to an ephemeral local disk
+
+You can move etcd from a root volume (Cinder) to a dedicated ephemeral local disk to prevent or resolve performance issues.
+
+## Prerequisites
+
+* An OpenStack cloud with Cinder being functional and enough available storage (at least 75 GB) to accommodate 3 Root Volumes for the OpenShift control plane.
+* An OpenStack cloud whith Nova not using Ceph as a backend for ephemeral disks.
```

Comment:
This should be moved to top as I think this is the number 1 requirement. Also, perhaps it is clearer if we don't use negative:
```suggestion
* An OpenStack cloud where Nova is configured to use local storage for ephemeral disks. The `libvirt.images_type` option in `nova.conf` must not be `rbd`.
```